Antenna_Performance_Metric/rootAnalysis.py

```diff
@@ -27,11 +27,12 @@
 
 
 
-lib_dir = '/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/'
-# lib_dir = '/users/PAS1960/dylanwells1629/buildingPueoSim/pueoBuilder/lib/'
+#lib_dir = '/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/'
+lib_dir = '/users/PAS1960/dylanwells1629/buildingPueoSim/may13pueo/lib/' #updated pueoSim path as of July 8th
```

Comment:
Minor style fixes:
```suggestion
# lib_dir = '/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/'
lib_dir = '/users/PAS1960/dylanwells1629/buildingPueoSim/may13pueo/lib/'  # updated pueoSim path as of July 8th
```

Also why not adding this to the arg parser above?
If you want a default you could do something like:
```py
DEFAULT_LIB_DIR = '/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/'
...
parser.add_argument("--lib-dir", default=DEFAULT_LIB_DIR,
                    help="The lib dir", type=str)
...
```